.pre-commit-config.yaml

```diff
@@ -237,3 +231,15 @@ repos:
         additional_dependencies:
         - flake8==4.0.1
         - flake8-pyi==22.5.1
+    -   id: future-annotations
+        name: import annotations from __future__
+        entry: 'from __future__ import annotations'
+        language: pygrep
+        args: [--negate]
+        files: ^pandas/
+        types: [python]
```

Comment:
`python` excludes *.pyi files, or? pyi files wouldn't need this future import (https://github.com/PyCQA/flake8-pyi Y044)

Reply:
yeah `python` is just Python files, `pyi` files have their own type (you can use `identify` to check):
```
$ identify-cli pandas/_libs/ops.pyi
["file", "non-executable", "pyi", "text"]
```